src/styles/index.css

```diff
@@ -0,0 +1,188 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: rgb(6, 11, 53);
+}
+
+.page__body {
+  margin: 0;
+}
+
+.nav {
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__list :not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  text-decoration: none;
+  color: rgb(6, 11, 53);
+  padding-block: 23px;
+  transition: 300ms;
+}
+
+.nav__link--hover:hover {
+  color: rgb(0, 172, 220);
+}
+
+.nav__link--active {
+  color: rgb(0, 172, 220);
+}
+
+.nav__link--active::before {
+  content: '';
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: rgb(0, 172, 220);
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header__logo-link {
+  display: flex;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--active :not(:last-child) {
+  background-image: url(/src/images/star-active.svg);
```

Comment:
The 'background-image' URL for '.stars--active' is set to '/src/images/star-active.svg'. Ensure that the path is correct and the image is available at the specified location.